cpp/test_python_call_cpp.py

```diff
@@ -60,6 +64,30 @@ def test_cross_language_cpp():
     assert students == ray.get(obj9)
 
 
+def test_cross_language_cpp_actor():
+    actor = ray.cross_language.cpp_actor_class("CreateCounter", "Counter").remote()
+    obj = actor.Plus1.remote()
+    assert 1 == ray.get(obj)
+
+    actor1 = ray.cross_language.cpp_actor_class(
+        "RAY_FUNC(Counter::FactoryCreate)", "Counter"
+    ).remote()
```

Comment:
Can you add tests for creating invalid actors?

Reply:
Sure